az-snp-vtpm/Makefile

```diff
@@ -39,4 +35,7 @@ delete:
 		--yes && \
 	az network public-ip delete \
 		--resource-group $(CVM_RESOURCE_GROUP) \
-		--name $(VM_NAME)-ip
+		--name $(VM_NAME)-ip && \
```

Comment:
Can we make deletion idempotent and resilient? Using `&&` makes it prone frequent failures.

Reply:
I think the az cli works like this ootb, if you delete a non-existing resource exit code will be still 0

Reply:
But for the failed command the chain will stop isn't it? Leaving dangling resources?

Reply:
Which command do you expect to fail? I think none of the delete commands should fail (unless maybe we can't talk to the api or sth), even if the resources do not exists.

regarding the deletion of the arm template. afaik that's not possible, it's unlike terraform or cloudformation as there is no state beyond deployment.

Reply:
Alright, I guess let's revisit if we encounter these problems later.